internal/rbd/rbd_util.go

```diff
@@ -694,7 +694,7 @@ func (ri *rbdImage) trashRemoveImage(ctx context.Context) error {
 	_, err = ta.AddTrashRemove(admin.NewImageSpec(ri.Pool, ri.RadosNamespace, ri.ImageID))
 
 	rbdCephMgrSupported := isCephMgrSupported(ctx, ri.ClusterID, err)
-	if rbdCephMgrSupported && err != nil {
+	if rbdCephMgrSupported && err != nil && !errors.Is(err, librbd.ErrNotFound) {
```

Comment:
@nixpanic is `AddTrashRemove` Mrg API is also support this error message?

Reply:
yes, it seemed so. I have been testing with the Ceph main branch, maybe things changed there?